docs/grading/attendance.md

```diff
@@ -29,6 +29,7 @@ Excused absences can be given for any of the following:
 * Conferences
 * External hackathons
 * Schedule conflicts with courses, exams, or extracurriculars
+  * In order to receive an excused absence, this conflict must take place during your scheduled large or small group session.
```

Comment:
Varun suggested that we were too lenient with excused absences when grading --- people were requesting excused absences for the Friday meeting before a Monday exam or for office hours